server/src/services/storage-template.service.spec.ts

```diff
@@ -163,6 +164,47 @@ describe(StorageTemplateService.name, () => {
         originalPath: newMotionPicturePath,
       });
     });
+    it('Should use handlebar if condition for album', async () => {
+      const asset = assetStub.image;
+      const user = userStub.user1;
+      const album = albumStub.oneAsset;
+      const config = structuredClone(defaults);
+      config.storageTemplate.template = '{{y}}/{{#if album}}{{album}}{{else}}other/{{MM}}{{/if}}/{{filename}}';
+      SystemConfigCore.create(systemMock, loggerMock).config$.next(config);
+
+      userMock.get.mockResolvedValue(user);
+      assetMock.getByIds.mockResolvedValueOnce([asset]);
+      albumMock.getByAssetId.mockResolvedValueOnce([album]);
+
+      expect(await sut.handleMigrationSingle({ id: asset.id })).toBe(JobStatus.SUCCESS);
+
+      expect(moveMock.create).toHaveBeenCalledWith({
+        entityId: asset.id,
+        newPath: `upload/library/${user.id}/${asset.fileCreatedAt.getFullYear()}/${album.albumName}/${asset.originalFileName}`,
+        oldPath: asset.originalPath,
+        pathType: AssetPathType.ORIGINAL,
+      });
+    });
+    it('Should use handlebar else condition for album', async () => {
+      const asset = assetStub.image;
+      const user = userStub.user1;
+      const config = structuredClone(defaults);
+      config.storageTemplate.template = '{{y}}/{{#if album}}{{album}}{{else}}other//{{MM}}{{/if}}/{{filename}}';
```

Comment:
I believe there's a `/` too much after `other`, right? (not that it really matters but still)

Reply:
Yes, indeed, that is intended to also test / confirm that any more than one / gets removed.